packages/atomic/package.json

```diff
@@ -41,6 +41,7 @@
     "@coveo/headless": "^1.13.0",
     "coveo-styleguide": "^9.26.0",
     "dayjs": "^1.10.4",
+    "dompurify": "^2.3.0",
```

Comment:
Is this complementary to escape-html? or was escape-html not sufficient?

Also comparing the bundle sizes, it's a lot heavier
https://bundlephobia.com/package/dompurify@2.3.0
https://bundlephobia.com/package/escape-html@1.0.3

Reply:
They are unrelated. `dompurify` filters elements to prevent XSS injections in `innerHTML`.